examples/globe_wfs_extruded.html

```diff
@@ -186,67 +186,6 @@
                 }
             });
 
-            function configPointMaterial(result) {
```

Comment:
Why you remove the point layer ?

Reply:
It is no longer available in the current API key, and it doesn't really add something to this example.